zulipterminal/core.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, config_file: str, theme: ThemeSpec,
         self.autohide = autohide
         self.notify_enabled = notify
         self.footlinks_enabled = footlinks
+        self.message_search = None
+        self.stream_button = None
```

Comment:
Adding this here kind of feels inappropriate , Would love any suggestions regarding the same .